src/controllers/main-controller.ts

```diff
@@ -4,26 +4,29 @@ import { alias } from 'drizzle-orm/pg-core';
 import { Router, Request, Response } from 'express';
 import Joi from 'joi';
 import _ from 'lodash';
+import { bignumber } from 'mathjs';
 
-import { DEFAULT_CACHE_TTL } from 'config/constants';
+import { DECIMAL_PLACES, DEFAULT_CACHE_TTL } from 'config/constants';
```

Comment:
```suggestion
import { DEFAULT_CACHE_TTL } from 'config/constants';
```
`DECIMAL_PLACES` is not used.